templates/template-sycamore/src/app.rs.lte

```diff
@@ -16,46 +16,49 @@ struct GreetArgs<'a> {
 }
 
 #[component]
-pub fn App<G: Html>(cx: Scope) -> View<G> {
-    let name = create_signal(cx, String::new());
-    let greet_msg = create_signal(cx, String::new());
+pub fn App() -> View {
+    let name = create_signal(String::new());
+    let greet_msg = create_signal(String::new());
 
-    let greet = move |e: Event| {
+    let greet = move |e: SubmitEvent| {
         e.prevent_default();
-        spawn_local_scoped(cx, async move {
+        spawn_local_scoped(async move {
             // Learn more about Tauri commands at {% if v2 %}https://tauri.app/develop/calling-rust/{% else %}https://v1.tauri.app/v1/guides/features/command{% endif %}
-            let args = serde_wasm_bindgen::to_value(&GreetArgs { name: &name.get() }).unwrap();
+            let args = serde_wasm_bindgen::to_value(&GreetArgs {
+				name: &name.get()
```

Comment:
looks like the new version `.get` requires `Copy` trait on the signal value, which seems weird but looking over the docs we need to use either `.with` or `.with_untracked`

Reply:
Sorry forgot about that. Should be fixed now.